### ISSUE TITLE
Grids groupShowSummaryOnHide="[true]" is not working

### ISSUE THREAD

**Issue by afattahi54** [struts-community-plugins/struts2-jquery#40]
It seems that setting the groupShowSummaryOnHide="[true]" is not working and we face free maker exception

```
FreeMarker template error (HTML_DEBUG mode; use RETHROW in production!)


For "#if" condition: Expected a boolean, but this evaluated to a string+extended_hash (String wrapped into f.e.b.StringModel):
==> parameters.groupShowSummaryOnHide?default(false)  [in template "template/jquery/grid-close.ftl" at line 138, column 16]

----
FTL stack trace ("~" means nesting-related):
    - Failed at: #if parameters.groupShowSummaryOnHide...  [in template "template/jquery/grid-close.ftl" at line 138, column 11]
----

Java stack trace (for programmers):
----
freemarker.core.NonBooleanException: [... Exception message was already printed; see it above ...]
    at freemarker.core.Expression.modelToBoolean(Expression.java:136)
    at freemarker.core.Expression.evalToBoolean(Expression.java:119)
    at freemarker.core.Expression.evalToBoolean(Expression.java:110)
    at freemarker.core.ConditionalBlock.accept(ConditionalBlock.java:46)
    at freemarker.core.Environment.visit(Environment.java:312)
    at freemarker.core.MixedContent.accept(MixedContent.java:62)
    at freemarker.core.Environment.visitByHiddingParent(Environment.java:333)
    at freemarker.core.ConditionalBlock.accept(ConditionalBlock.java:48)
    at freemarker.core.Environment.visit(Environment.java:312)
    at freemarker.core.MixedContent.accept(MixedContent.java:62)
    at freemarker.core.Environment.visit(Environment.java:312)
    at freemarker.core.Environment.process(Environment.java:290)
    at freemarker.template.Template.process(Template.java:312)
    at org.apache.struts2.components.template.FreemarkerTemplateEngine.renderTemplate(FreemarkerTemplateEngine.java:158)


```

**Comment by aleksandr-m** [comment]
Have you tried `groupShowSummaryOnHide="true"`?

**Comment by afattahi54** [comment]
Yes no luck!

**Comment by flofourcade** [comment]
Wrong handling of this parameter in Grid.java. Fixed it, will be OK in 4.0.3.

NB the right syntax will be `groupShowSummaryOnHide="true"`

**Comment by jogep** [comment]
Close issue because version 4.0.3 is released.
